docs/source/modules/zos_blockinfile.rst

```diff
@@ -33,7 +33,7 @@ src
 
   The USS file must be an absolute pathname.
 
-  Generation data set (GDS) relative name of generation already created.  ``e.g. SOME.CREATION(-1``.)
+  Generation data set (GDS) relative name of generation already created. e.g. *SOME.CREATION(-1*).
```

Comment:
similar comment as before, seems like the '*' should be after the close paren ')'

Reply:
Same as above comment only different module 
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/20999c98-2df2-47b1-8e78-9cd0657a86fd">